notebooks/notebook_template.ipynb

```diff
@@ -43,7 +43,7 @@
         "    </a>\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    ! pip3 install --upgrade tensorflow
</pre><p>Can't you just use Tensorflow as one of the example dependencies to install earlier in the notebook.?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/572/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='44'/>